### ISSUE TITLE
Error in parsing F5 configuration. (junos parser)

### ISSUE THREAD

**Issue by cpaillet** [mpenning/ciscoconfparse#49]
Hi,

I think you have a issue when open and close curly brackets are at the same line.
When I parse 

```
ltm virtual virtual1 {
    profiles {
        test1 { }
    }
}
ltm virtual virtual2 {
    profiles2 {
        test2 { }
    }
}
```

profiles2 don't have any parent

**Comment by red-bin** [comment]
For anyone trying to parse this in the future, I fixed this by doing:
lb_config_fh = config_fh = open('a5.conf,'r')
lb_lines = [ line.replace('}','\n }') for line in lb_config_fh.readlines() ]
lb_lines = ''.join(lb_lines).replace('\n\n','\n')
lb_lines = lb_lines.split('\n')
parse(lb_lines, syntax='junos', comment='#')

<CiscoConfParse: 1243 lines / syntax: junos / comment delimiter: '#' / factory: False>

**Comment by mpenning** [comment]
This should be fixed in version 1.5.1
